ipatests/test_integration/test_adtrust_install.py

```diff
@@ -276,6 +278,21 @@ def test_ipa_user_pac(self):
             tasks.kinit_admin(self.master)
             self.master.run_command(['ipa', 'user-del', user])
 
+    @pytest.mark.xfail(
+        osinfo.id == 'fedora',
+        reason='Test requires slapi-nis with fix', strict=True)
```

Comment:
would be good if required slapi-nis version is mentioned OR you can check for slapi-nis version in the test and skip/xfail depending upon version.

Reply:
Added check for version and marked as xfail.